NOTICE.txt

```diff
@@ -9,9 +9,35 @@ Third party libraries used by the Elastic Beats project:
 ==========================================================================
 
 
+--------------------------------------------------------------------
+Dependency: 4d63.com/embedfiles
+Revision: 995e0740726f
+License type (autodetected): MIT
+./vendor/4d63.com/embedfiles/LICENSE:
+--------------------------------------------------------------------
+Copyright (c) 2017, Leigh McCulloch
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
 --------------------------------------------------------------------
 Dependency: 4d63.com/tz
-Revision: 6d37baae851b6f07225308b199322f70907aa3dd
+Revision: 6d37baae851b
```

Comment:
Is it possible to keep the complete hash?

Reply:
The problem is that go modules only store the short hash of a revision both in `go.mod` and `modules.txt`. Also, go downloads the deps into a simple folder, not as a git repository, so I cannot get the long version from there.
Alternatively, I can fetch all repos and get the long version of each hash using `git rev-parse` but it felt like overkill for a NOTICE. WDYT?

Reply:
I see.